bears/stylus/StylintBear.py

```diff
@@ -0,0 +1,36 @@
+from coalib.bearlib.abstractions.Linter import linter
+from dependency_management.requirements.NpmRequirement import NpmRequirement
+
+
+@linter(executable='stylint',
+        output_format='regex',
+        output_regex=r'(?P<line>\d+)[:]*(?P<column>\d+)*\s*.*'
```

Comment:
Can you turn `\s*` into `\s+`? I assume that there will always be at least one space^^